panacea/query_client.go

```diff
@@ -319,11 +320,19 @@ func (q QueryClient) GetLightBlock(height int64) (*tmtypes.LightBlock, error) {
 	return q.lightClient.TrustedLightBlock(height)
 }
 
-func (q QueryClient) GetOracleParamsPublicKey() ([]byte, error) {
+func (q QueryClient) GetOracleParamsPublicKey() (*btcec.PublicKey, error) {
 	oraclePubKeyBz, err := q.GetStoreData(context.Background(), paramstypes.StoreKey, append(append([]byte(oracletypes.StoreKey), '/'), oracletypes.KeyOraclePublicKey...))
 	if err != nil {
 		return nil, err
 	}
+	if oraclePubKeyBz == nil {
```

Comment:
For now, I just put an error handling for nil value of oracle public key when querying oracle params in panacea-core.